src/Text/Pandoc/Readers/LaTeX.hs

```diff
@@ -2070,12 +2070,11 @@ addImageCaption :: PandocMonad m => Blocks -> LP m Blocks
 addImageCaption = walkM go
   where go (Image attr@(_, cls, kvs) alt (src,tit))
             | not ("fig:" `T.isPrefixOf` tit) = do
-          mbcapt <- sCaption <$> getState
-          mblabel <- sLastLabel <$> getState
-          let (alt', tit') = case mbcapt of
+          mbs <- getState
+          let (alt', tit') = case sCaption mbs of
```

Comment:
mbs seems an odd namefor this.  Why not st ?
And is there any advantage to doing it this way?  I would be surprised if there's a measurable performance difference.

Reply:
I picked `mbs` because the style here looked like `mb` followed by an abbreviation of what it was, so I tried to follow it with `s` for "state". I'm fine with `st` instead if you want. As for speed, it will be slightly faster, since you're basically replacing `m >>= return . f >>= g` with `m >>= g . f`. It's definitely not a big difference, though. My main reason for the change was to reduce cognitive load.

Reply:
I would have thought the compiler would have optimized this difference away, but maybe not.
Let's use `st` instead of `mbs` - that's usually what I use for the state variable.

Reply:
I don't think the compiler does optimizations that require that instances be lawful to be correct (though I could be wrong, maybe if enough stuff gets specialized). Anyway, I just changed it to `st`.